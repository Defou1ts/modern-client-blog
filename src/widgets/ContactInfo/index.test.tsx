import React from 'react';

import { render, screen } from '@testing-library/react';

import renderer from 'react-test-renderer';

import { ContactInfo } from './index';

describe('ContactInfo', () => {
	it('renders a  ContactInfo', () => {
		render(<ContactInfo />);
		const text = screen.getByText('contact.title');
		expect(text).toBeInTheDocument();
		expect(text).toBeVisible();
	});

	it('ContactInfo snapshot', () => {
		const component = renderer.create(<ContactInfo />);

		const tree = component.toJSON();
		expect(tree).toMatchSnapshot();
	});
});
