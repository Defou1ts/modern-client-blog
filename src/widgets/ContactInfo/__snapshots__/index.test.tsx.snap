// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContactInfo ContactInfo snapshot 1`] = `
<section
  className="wrapper"
>
  <h6
    className="h h6 contact"
  >
    contact.title
  </h6>
  <h1
    className="h h1 conversation"
  >
    contact.subtitle
  </h1>
  <p
    className="p medium description"
  >
    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim.
  </p>
</section>
`;
